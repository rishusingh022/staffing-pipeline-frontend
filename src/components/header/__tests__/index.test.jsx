import React from 'react';
import { render } from '@testing-library/react';
import Header from '..';

describe('Header', () => {
  it('should render correctly with a navbar when Navbar boolean is true', () => {
    const { asFragment } = render(<Header {...{ hasNav: true }} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render correctly without a navbar when Navbar boolean is false', () => {
    const { asFragment } = render(<Header {...{ hasNav: false }} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
