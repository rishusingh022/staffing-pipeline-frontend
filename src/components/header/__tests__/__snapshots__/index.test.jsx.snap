// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly with a navbar when Navbar boolean is true 1`] = `
<DocumentFragment>
  <div
    class="font-light h-[88px] text-white flex justify-center bg-deepBlue"
  >
    <div
      class="h-full xl:w-[63vw] lg:[70vw] md:[80vw] flex box-border"
    >
      <img
        class="h-[70px] self-end pl-0 lg:pr-[180px] sm:pr-[90px] pt-0 pb-[5px]"
        src="./assets/McK_Logo.png"
      />
      <div
        class="grid grid-cols-3 gap-10 items-end"
      >
        <button
          class="text-[white] hover:text-cyan hover:border-b-[5px] hover:border-b-cyan hover:border-solid pt-0 pb-[15px] hover:pb-2.5 px-[5px]"
        >
          Engagements
        </button>
        <button
          class="text-[white] hover:text-cyan hover:border-b-[5px] hover:border-b-cyan hover:border-solid pt-0 pb-[15px] hover:pb-2.5 px-[5px]"
        >
          People
        </button>
        <button
          class="text-[white] hover:text-cyan hover:border-b-[5px] hover:border-b-cyan hover:border-solid pt-0 pb-[15px] hover:pb-2.5 px-[5px]"
        >
          Case Studies
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Header should render correctly without a navbar when Navbar boolean is false 1`] = `
<DocumentFragment>
  <div
    class="font-light h-[88px] text-white flex justify-center bg-deepBlue"
  >
    <div
      class="h-full xl:w-[63vw] lg:[70vw] md:[80vw] flex box-border"
    >
      <img
        class="h-[70px] self-end pl-0 lg:pr-[180px] sm:pr-[90px] pt-0 pb-[5px]"
        src="./assets/McK_Logo.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
